apps/prairielearn/assets/scripts/instructorAssessmentInstancesClient.ts

```diff
@@ -158,16 +171,97 @@ onDocumentReady(() => {
   });
 
   function tableColumns(assessmentGroupWork: boolean) {
-    const assessmentInstanceIdColumn = {
-      field: 'assessment_instance_id',
-      title: '<span class="sr-only">Assessment Instance</span>',
-      sortable: true,
-      sorter: detailsLinkSorter,
-      formatter: detailsLinkFormatter,
-      class: 'align-middle sticky-column text-nowrap',
-      switchable: false,
-    };
-    const nonspecificColumns = [
+    return [
+      {
+        field: 'assessment_instance_id',
+        title: '<span class="sr-only">Assessment Instance</span>',
+        sortable: true,
+        sorter: detailsLinkSorter,
+        formatter: detailsLinkFormatter,
+        class: 'align-middle sticky-column text-nowrap',
+        switchable: false,
+      },
+      ...(assessmentGroupWork
+        ? [
+            {
+              field: 'group_name',
+              title: 'Name',
+              visible: false,
+              sortable: true,
+              class: 'align-middle',
+              switchable: true,
+              formatter: (value: string) => html`${value}`.toString(),
```

Comment:
Does this work if you use `escapeHtml(...)` instead? It should do the same thing, but be more explicit about why it's being done.

Reply:
`escapeHtml()` receives an `HtmlSafeString` as parameter and returns an `HtmlSafeString`. Converting the return value of `escapeHtml` back to a string would escape it twice.

Reply:
That's not true, you can prove this to yourself with this little snippet:

```js
import { escapeHtml } from '@prairielearn/html';
console.log(escapeHtml('<script>alert("hi")</script>').toString());
```

That logs the following, which is not double-escaped:

```
&lt;script&gt;alert(&#34;hi&#34;)&lt;/script&gt;
```

Reply:
You are passing a string as the first argument, which doesn't match the function signature. This will trigger the type checker.

Reply:
Indeed 🙃 

Maybe we should export `escapeHtmlRaw` from that library? We can do that in a future PR though.